src/__tests__/JoinTheSquadContainer.test.js

```diff
@@ -12,15 +12,11 @@ const store = mockStore();
 
 afterEach(cleanup);
 
-jest.mock('@okta/okta-react', () => ({
-  useOktaAuth: () => {
+jest.mock('@auth0/auth0-react', () => ({
+  useAuth0: () => {
     return {
-      authState: {
-        isAuthenticated: true,
-      },
-      authService: {
-        getUser: () => Promise.reject(),
-      },
+      authState: jest.fn(),
```

Comment:
Should this be user instead of authState?

Reply:
Yes I'll correct that now.